wpiunits/src/main/java/edu/wpi/first/units/AccelerationUnit.java

```diff
@@ -54,15 +54,15 @@ public Acceleration<D> ofBaseUnits(double baseUnitMagnitude) {
   }
 
   @Override
-  @SuppressWarnings("unchecked")
-  public Measure<AccelerationUnit<D>> zero() {
-    return (Measure<AccelerationUnit<D>>) super.zero();
+  @SuppressWarnings({"unchecked", "rawtypes"})
+  public Acceleration<D> zero() {
+    return (Acceleration<D>) (Acceleration) super.zero();
```

Comment:
A raw cast is required here due to conflicts with the wildcarded return supertype:

```
wpiunits/src/main/java/edu/wpi/first/units/AccelerationUnit.java:59: error: incompatible types: Measure<CAP#1> cannot be converted to Acceleration<D>
    return (Acceleration<D>) super.zero();
                                       ^
  where D is a type-variable:
    D extends Unit declared in class AccelerationUnit
  where CAP#1 is a fresh type-variable:
    CAP#1 extends PerUnit<VelocityUnit<D>,TimeUnit> from capture of ? extends PerUnit<VelocityUnit<D>,TimeUnit>
```